src/main/java/seedu/stock/logic/parser/AddCommandParser.java

```diff
@@ -37,7 +37,7 @@ public AddCommand parse(String args) throws ParseException {
         }
 
         Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
-        SerialNumber serialNumber = SerialNumber.generateSerialNumber();
+        SerialNumber serialNumber = SerialNumber.generateDefaultSerialNumber();
```

Comment:
Would it be better if there was no need to generate default serial number here? As I feel it is an extra step before we get the actual serial number for the stock. I may have an idea of how to do it, and can help to work on it!

Reply:
Where do you suggest then? The other location I can think of is stock class.